src/pypromice/process/resample.py

```diff
@@ -34,6 +34,15 @@ def resample_dataset(ds_h, t):
     '''
     df_d = ds_h.to_dataframe().resample(t).mean()
     
+    # taking the 10 min data and using it as instantaneous values:
+    if (t == '60min') and (ds_h.time.diff(dim='time').isel(time=0).dt.total_seconds() == 600):
```

Comment:
Isn't it possible to check if it is a raw dataset more directly than looking at the first time diff? Like `dataset.attrs["format"]`

Reply:
Indeed we could, but I spotted last week files that were entered as "raw" instead of "STM" in the config files, causing processing errors down the line.
So I figured I wouldn't rely on a typed-in info.